### DISCUSSION TITLE
Enhancement request: Option to always create closing tags

### DISCUSSION THREAD

**Discussion by mfsasso** [NaturalIntelligence/fast-xml-parser#474] [Ideas]
I have an XML file that contains some empty tags, represented as _\<x\>\</x\>_ (where 'x' is whatever). My file passes validation, so as far as I know it's okay to write XML that way. And I would like to preserve that format. However, if I run my file through the parser and then through the builder, the output for those tags is _\<x/\>_. Can an option be added to suppress the use of self-closing tags for empty data?

**Comment by amitguptagwl** [comment]
As I remember, a good discussion happened on this requirement and some solution was provided. Check `suppressEmptyNode` and `suppressUnpairedNode` from the docs. I hope it should help
